Applications/Express.md

```diff
@@ -0,0 +1,76 @@
+## Express
+
+The express project is applying to become an incubating
+[Top Level Project of the Node.js Foundation][].
+
+### About Express, User base, Community and Ecosystem
+
+Express is one of the most popular web server frameworks available for Node.js.
+For many Node.js developers, express is their primary interface with Node.js
+on the server.
+
+Express has consistently been among the top packages installed via npm, with
+nearly 4.5 million downloads in just the past month.
+
+It is important to recognize that Express is not the only web framework in the
+Node.js ecosystem and that by incubating Express, neither the Node.js
+Foundation or Node.js core would be "picking a winner" or endorsing Express
+over any other overlapping or similar project.
+
+### Formation of an Express WG
+
+The express project consists of a single top level core repository
+(strongloop/express) that has dependencies on a large number of smaller
+independent components that are currently spread out across multiple
+independent GitHub repositories. This proposal moves only the
+strongloop/express GitHub repository under the Node.js GitHub organization
+(nodejs/express).
+
+If the proposal for incubating express as a top level project is accepted, a
+call for participation would be put out to existing Node.js GitHub organization
+members and an "Express WG" (nodejs/express team in the github organization)
+with commit rights to the nodejs/express repository would be created. This WG
+would also initially include existing contributors to the strongloop/express
+github repository who are not currently members of the Node.js GitHub
+organization.
+
+A secondary nodejs/express-release team would be created consisting of the
+subset of collaborators directly involved in the release process. These would
+be identified out of the members of the main Express WG (following the
+precedence of the nodejs/build team).
+
+While under incubation, there would be no formal "Express TC". If and when a
+decision is made to charter the express project as a full top level project,
+and if a core group of technical contributors has emerged out of the base of
+collaborators, a formal TC can be established as part of the express TLP
+charter. Initially, however, the bias is towards growing the base of
+contributors as opposed to establishing or imposing a governing body.
+
+Essentially, the "TC" will be whoever steps up to do the necessary work.
+
+### Contribution and Governance Process
+
+The contribution process would be the same lazy consensus model used by Node.js
```

Comment:
A more accurate definition is "lazy consensus *seeking* model."

It basically works like this:
* All contributions go through review and try to move towards the obvious consensus.
* The default is for things to land given that nobody objects. There's no need for a vote or formal process around getting +1's for every change, you propose the change, make any changes you need to in the review process, and land the change if nobody is against it.
* When consensus can't be reached easily it escalates to a more formal process for discussion (usually a weekly call). This is commonly less than 5% of all contributions, most just get reviewed and land.
* If a consensus can't be reached then a vote is taken, majority wins. In reality, the fact that a vote *could* happen is enough incentive to work towards a consensus and for people to drop objections they have that don't have wider support. Node.js Core has never had to bring an issue to a formal vote but the fact that a vote is a possibility keeps anyone from holding up the process with their own objection without trying to widen their support to other contributors. In a "pure consensus" model that's a huge problem, everyone essentially has a veto and aren't incentivized to try and convince their peers, so we avoid that with a "consensus seeking" system.